presto-manager-common/src/main/java/com/teradata/prestomanager/common/JaxrsParameter.java

```diff
@@ -39,6 +39,8 @@ public JaxrsParameter(String s)
     protected abstract T parseString(String s)
             throws ParseException;
 
+    protected abstract String getString();
```

Comment:
`null` is a poor string representation.

It might make more sense to return an `Optional<String>`. Then it might also make sense to change the behavior of `get()` to behave like `Optional::get`.